github_actions/spec/dependabot/github_actions/file_parser_spec.rb

```diff
@@ -215,6 +215,14 @@ def mock_service_pack_request(nwo)
       end
     end
 
+    describe "with a local reusable workflow that has an @ in the file name" do
+      let(:workflow_file_fixture_name) { "local_workflow_with_at.yml" }
```

Comment:
I'm not clear on why the `@` has significance to us in this scenario. My understanding of the change, and the behavior it's fixing, is that the locality (i.e. the `.`) is more significant. We wouldn't attempt to update `./.github/workflows/test.yml@2.1.0` because it's co-located in the repo. Am I reading that correctly? If so, I think it makes sense to leave off the `@` detail as it suggests the crux of the issue is with the version. What do you think?

Reply:
Fair enough, it's just that the `@` is what was causing it to be an issue.

Reply:
> Fair enough, it's just that the @ is what was causing it to be an issue.

@jakecoffman By this, do you mean that the `@` was an issue for this particular user? Or that it's an issue that's widely seen? I definitely don't want to push you away from your instincts/what you think is right!

I guess another way to ask this would be: what would Dependabot do if the `@2.1.0` was missing?

Reply:
If there was no @ then there was no bug, it just happened to work I think. The @ made the bug come out because of [this regex](https://github.com/dependabot/dependabot-core/blob/742ba48c115295042564ba97bfd5d3af82d38033/github_actions/lib/dependabot/github_actions/file_parser.rb#L23-L24). 

In the end though, the @ isn't important. The important thing is we're ignoring paths as dependencies because Dependabot can't update them.